tas-env-variables.sh

```diff
@@ -1,25 +1,19 @@
 #!/bin/bash
 
-export BASE_HOSTNAME=apps.$(oc get dns cluster -o jsonpath='{ .spec.baseDomain }')
-echo "base hostname = $BASE_HOSTNAME"
+# This assumes you are currently running from the context of the namespace where your securesign is created
+# Run `oc project <securesign namespace>` to ensure you are working within the correct context
 
-
-export KEYCLOAK_REALM=sigstore
-export KEYCLOAK_URL=https://keycloak-keycloak-system.$BASE_HOSTNAME
-export TUF_URL=https://tuf.$BASE_HOSTNAME
-export COSIGN_FULCIO_URL=https://fulcio.$BASE_HOSTNAME
-export COSIGN_REKOR_URL=https://rekor.$BASE_HOSTNAME
+export TUF_URL=$(oc get tuf -o jsonpath='{.items[0].status.url}')
+export OIDC_ISSUER_URL=https://$(oc get route keycloak -n keycloak-system | tail -n 1 | awk '{print $2}')/auth/realms/trusted-artifact-signer
+export COSIGN_FULCIO_URL=$(oc get fulcio -o jsonpath='{.items[0].status.url}')
+export COSIGN_REKOR_URL=$(oc get rekor -o jsonpath='{.items[0].status.url}')
 export COSIGN_MIRROR=$TUF_URL
 export COSIGN_ROOT=$TUF_URL/root.json
-export COSIGN_OIDC_ISSUER=$KEYCLOAK_URL/auth/realms/$KEYCLOAK_REALM
-export COSIGN_CERTIFICATE_OIDC_ISSUER=$COSIGN_OIDC_ISSUER
+export COSIGN_OIDC_ISSUER=$OIDC_ISSUER_URL
+export COSIGN_OIDC_CLIENT_ID=trusted-artifact-signer
```

Comment:
I don't know how important the go binary installer is to GA, but I remember there being issues for TP2 with theses variables not being up-to-date with https://github.com/securesign/sigstore-ocp/blob/release-1.0.gamma/tas-installer/cmd/envgen.go#L33

So they may need an update too

Reply:
I updated the script `sigstore` -> `trusted-artifact-signer` and also added the CLIENT_ID env var there